src/scitokens/tools/admin_create_token.py

```diff
@@ -22,6 +22,8 @@ def add_args():
                         help='Location of the private key file')
     parser.add_argument('--key_id', help='The string key identifier')
     parser.add_argument('--issuer', help="Issuer for the token")
+    parser.add_argument('--lifetime', help="Lifetime of the token, in seconds from now. Default: 1200 seconds, 20 minutes",
```

Comment:
```suggestion
    parser.add_argument('--lifetime', help="Lifetime of the token, in seconds from now. Default: 1200 seconds (20 minutes)",
```

Reply:
Being picky... just so it's clear it's not 1200 seconds + 20 minutes. (If you just want to list seconds, you could use `%(default)s seconds` to pull in the default value)